site2/docs/concepts-messaging.md

```diff
@@ -61,11 +61,11 @@ Producers send messages to brokers synchronously (sync) or asynchronously (async
 
 You can have different types of access modes on topics for producers.
 
-|Access mode | Description
-|---|---
-`Shared`|Multiple producers can publish on a topic. <br><br>This is the **default** setting.
-`Exclusive`|Only one producer can publish on a topic. <br><br>If there is already a producer connected, other producers trying to publish on this topic get errors immediately.<br><br>The “old” producer is evicted and a “new” producer is selected to be the next exclusive producer if the “old” producer experiences a network partition with the broker.
-`WaitForExclusive`|If there is already a producer connected, the producer creation is pending (rather than timing out) until the producer gets the `Exclusive` access.<br><br>The producer that succeeds in becoming the exclusive one is treated as the leader. Consequently, if you want to implement the leader election scheme for your application, you can use this access mode.
+| Access mode        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Table changes (auto fix by IntelliJ) to match GitHub Markdown required (must start and end with pipe)